packages/typespec-test/test/todo_non_branded/spec/main.tsp

```diff
@@ -248,7 +249,7 @@ namespace TodoItems {
 
   @route("{itemId}/attachments")
   namespace Attachments {
-    @list op list(
+    op list(
```

Comment:
why we have this change?

Reply:
see this comment: https://github.com/microsoft/typespec/issues/5048#issuecomment-2467627787